src/Emogrifier.php

```diff
@@ -1030,6 +1021,20 @@ function (array $cssRule) {
         }
     }
 
+    /**
+     * @param string[] $cssRule
+     *
+     * @return bool
+     */
```

Comment:
This should have `@throws \InvalidArgumentException` in the PHPDoc.

Reply:
Done.